tests/nested/manual/core20-install-mode-shutdown-via-hook/task.yaml

```diff
@@ -0,0 +1,91 @@
+summary: Verify snapctl poweroff install-device hook usage
+
+systems: [ubuntu-20.04-64]
+
+environment:
+    NESTED_IMAGE_ID: core20-install-device
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+    NESTED_ENABLE_TPM: true
+    NESTED_ENABLE_SECURE_BOOT: true
+
+details: |
+    This test checks support for shutting down the device at the end of install
+    mode via the install-device hook
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    # needed for inspecting the partitions
+    snap install jq
```

Comment:
jq should be removed on restore, on nested the automatic clean up is not done

Reply:
removed now via tests.cleanup